test_fakeredis.py

```diff
@@ -4013,7 +4012,7 @@ class TestFakeRedis(unittest.TestCase):
 
     def setUp(self):
         if REDIS3:
-            raise SkipTest('Legacy redis class does not apply to redis-py 3+')
+            pytest.skip('Legacy redis class does not apply to redis-py 3+')
```

Comment:
If `redis3_only` is rewritten as a pytest marker then it can be used to decorate the class.